psse/psse-model/src/main/java/com/powsybl/psse/model/PsseContext.java

```diff
@@ -0,0 +1,176 @@
+/**
+ * Copyright (c) 2020, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.psse.model;
+
+/**
+ *
+ * @author Luma Zamarreño <zamarrenolm at aia.es>
+ * @author José Antonio Marqués <marquesja at aia.es>
+ */
+public class PsseContext {
```

Comment:
As nothing is public is that class, maybe this class should also be private package

Reply:
Done